docs/cody/enterprise/completions-configuration.mdx

```diff
@@ -0,0 +1,186 @@
+# Completions Configuration
+
+<p className="subtitle">Learn how to configure Cody via `completions` on a Sourcegraph Enterprise instance.</p>
+
+<Callout type="info">Configuring Cody via `completions` is legacy but it's still supported. We recommend using the new [`modelConfiguration`](/cody/enterprise/model-configuration) for flexible LLM model selection.</Callout>
+
+[Cody Enterprise](https://sourcegraph.com/enterprise) supports many models and model providers. You can configure Cody Enterprise to access models via Sourcegraph Cody Gateway or directly using your own model provider account or infrastructure. Let's look at these options in more detail.
+
+## Using Sourcegraph Cody Gateway
+
+This is the recommended way to configure Cody Enterprise. It supports all the latest models from Anthropic, OpenAI, Mistral, and more without requiring a separate account or incurring separate charges. You can learn more about these in our [supported models](/cody/capabilities/supported-models) docs.
+
+## Using your organization's account with a model provider
+
+### Example: Your organization's Anthropic account
+
+First, [create your own key with Anthropic](https://console.anthropic.com/account/keys). Once you have the key, go to **Site admin > Site configuration** (`/site-admin/configuration`) on your instance and set:
+
+```json
+{
+  // [...]
+  "cody.enabled": true,
+  "completions": {
+    "provider": "anthropic",
+    "chatModel": "claude-2.0", // Or any other model you would like to use
+    "fastChatModel": "claude-instant-1.2", // Or any other model you would like to use
+    "completionModel": "claude-instant-1.2", // Or any other model you would like to use
+    "accessToken": "<key>"
+  }
+}
+```
+
+### Example: Your organization's OpenAI account
+
+First, [create your own key with OpenAI](https://beta.openai.com/account/api-keys). Once you have the key, go to **Site admin > Site configuration** (`/site-admin/configuration`) on your instance and set:
+
+```json
+{
+  // [...]
+  "cody.enabled": true,
+  "completions": {
+    "provider": "openai",
+    "chatModel": "gpt-4", // Or any other model you would like to use
+    "fastChatModel": "gpt-3.5-turbo", // Or any other model you would like to use
+    "completionModel": "gpt-3.5-turbo-instruct", // Or any other model that supports the legacy completions endpoint
+    "accessToken": "<key>"
+  }
+}
+```
+
+[Learn more about OpenAI models.](https://platform.openai.com/docs/models)
+
+## Using your organization's public cloud infrastructure
+
+### Example: Use Amazon Bedrock
+
+You can use Anthropic Claude models on [Amazon Bedrock](https://aws.amazon.com/bedrock/).
```

Comment:
Nit: A half-sentence about _why_ one might do that would seem appropriate and helpful to me here.